### PR TITLE
Feature/5160 move reports

### PR DESCRIPTION
**Pull request by melsawy**
- Add projects list to `/me` endpoint
- Update ES after move media to other project
- Add tests

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -114,6 +115,14 @@ def user_teams
     teams.to_json
   end
 
+  def user_projects
+    projects = Hash.new
+    self.teams.each do |t|
```
Comment (caiosba) [inline app/models/user.rb]: Will this include teams that the user requested access but is not approved yet?

Comment (melsawy) [inline app/models/user.rb]: yes will change it

Comment (melsawy) [inline app/models/user.rb]: Done with a unit test

Comment (caiosba) [inline app/models/user.rb]: Thanks!

**Pull request merged**